### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by bvarjavand**
The readthedocs documentation is more appropriate for this package compared to a readme in the Github repository.

### CODE REVIEW

File: brainlit/utils/session.py [hunk 1]
```diff
@@ -135,6 +136,7 @@ def get_segments(self, seg_id: int, bbox: Optional[Bounds] = None) -> nx.Graph:
             G = get_sub_neuron(G, [bbox[:3], bbox[3:]])
         return G
 
+    # TOMMY REVIEW
     def create_tubes(
```
Comment (tathey1) [inline brainlit/utils/session.py:140]: something more like "Create voxel-wise foreground/background labels associated with a particular neuron trace within a given bounding box of voxel coordinates."

File: brainlit/utils/session.py [hunk 2]
```diff
@@ -111,6 +111,7 @@ def set_url_segments(self, seg_url: str):
         self.url_segments = seg_url
         self.cv_segments = CloudVolume(self.url_segments, parallel=False)
 
+    # TOMMY REVIEW
     def get_segments(self, seg_id: int, bbox: Optional[Bounds] = None) -> nx.Graph:
```
Comment (tathey1) [inline brainlit/utils/session.py:115]: whats the type of output graph? networkx?

File: brainlit/utils/session.py [hunk 3]
```diff
@@ -236,6 +240,7 @@ def pull_vertex_list(
         img = self.pull_bounds_img(bounds)
         return img, bounds, vox_in_img_list
 
+    # TOMMY REVIEW
     def pull_chunk(
```
Comment (tathey1) [inline brainlit/utils/session.py:244]: I think you mean, pull a "subvolume" of an image around a point, not "chunks"

File: brainlit/utils/session.py [hunk 4]
```diff
@@ -196,6 +199,7 @@ def pull_voxel(
         vox_in_img = voxel - np.array(bounds.to_list()[:3])
         return np.squeeze(np.array(img)), bounds, vox_in_img
 
+    # TOMMY REVIEW
     def pull_vertex_list(
```
Comment (tathey1) [inline brainlit/utils/session.py:203]: in the docs you say pull a "region." HOwever i've seen other places where you use other words like "subvolume" or "image." Go ahead and decide on a single word and use only it.

File: brainlit/utils/session.py (hunk 2 above)
Comment (bvarjavand) [reply brainlit/utils/session.py:115]: Yes, if you look at the type hints it is clear. I will add a brief descriptor.

File: brainlit/utils/session.py (hunk 3 above)
Comment (bvarjavand) [reply brainlit/utils/session.py:244]: No, I actually mean chunks

File: brainlit/utils/session.py (hunk 4 above)
Comment (bvarjavand) [reply brainlit/utils/session.py:203]: let's go with volume and subvolume

File: brainlit/utils/session.py (hunk 1 above)
Comment (bvarjavand) [reply brainlit/utils/session.py:137]: done

File: brainlit/algorithms/generate_fragments/tube_seg.py [hunk 5]
```diff
@@ -149,6 +149,7 @@ def tubes_seg(img, vertices, radius, spheres=False):
     return labels
 
 
+# TOMMY REVIEW
```
Comment (tathey1) [inline brainlit/algorithms/generate_fragments/tube_seg.py:152]: I like it

**Review by tathey1 — APPROVED** [review]

**Pull request merged**